app/models/alert.rb

```diff
@@ -157,7 +157,11 @@ def new_comments
     comments = []
     # Doing this in this roundabout way because I'm not sure how to use "near" together with joins
     applications_with_new_comments.each do |application|
-      comments += application.comments.visible.where('comments.confirmed_at > ?', cutoff_time)
+      comments += application
```

Comment:
👍 Is this because Postgres defaults to some other order?

I can't find an example in PlanningAlerts, but here's an example of our preferred style for wrapping methods over lines like this https://github.com/mysociety/alaveteli/pull/3544/files#diff-4676c008b11a5480d73d4a6de01e45b9R216

I think splitting this over lines if definitely an improvement in readability 👍